modules/i3s/src/lib/parsers/constants.ts

```diff
@@ -1,17 +1,17 @@
 import GL from '@luma.gl/constants';
-import {DATA_TYPE} from '../../types';
+import {I3SAttributeType} from '../../types';
 
 export function getConstructorForDataFormat(dataType: string) {
   switch (dataType) {
-    case DATA_TYPE.UInt8:
+    case I3SAttributeType.UInt8:
```

Comment:
Nit: Possible to just use strings? `type I3SAttributeType = 'uint8' | ... ;` The goal is to modernize the API away from enums where possible.